TestFoundation/TestNSProcessInfo.swift

```diff
@@ -53,4 +54,16 @@ class TestNSProcessInfo : XCTestCase {
         processInfo.processName = originalProcessName
         XCTAssertEqual(processInfo.processName, originalProcessName, "\"\(processInfo.processName)\" not equal to \"\(originalProcessName)\"")
     }
+    
+    func test_globallyUniqueString() {
+        let uuid = NSProcessInfo.processInfo().globallyUniqueString
+        let parts = uuid._bridgeToObject().componentsSeparatedByString("-")
```

Comment:
_bridgeToObject is the stub for the automatic type conversion from the compiler, if you need to bridge use the bridge() method